apps/builder/app/builder/features/style-panel/style-layers-list.tsx

```diff
@@ -26,27 +30,27 @@ export type LayerProps<T> = {
   disabled?: boolean;
   onLayerHide: (index: number) => void;
   onDeleteLayer: (index: number) => void;
-  onEditLayer: (index: number, layers: LayersValue) => void;
+  onEditLayer: (index: number, layers: LayersValue | TupleValue) => void;
   createBatchUpdate: CreateBatchUpdate;
   deleteProperty: DeleteProperty;
 };
 
-type LayerListProperties<T> = SectionProps & {
+type LayerListProperties<T, K> = SectionProps & {
```

Comment:
Please try to not use T,K when possible, e.g. in this case K is LayersValue  and T is TupleValue

Reply:
or maybe I am wrong, haven't looked exactly and T K isn't makeit it obvious